### PR TITLE
Add missing German and Dutch translation

### PR DESCRIPTION
**Pull request by 3d-gussner**
menu in German is Menü

Found the missing translation looking at https://github.com/prusa3d/Prusa-Firmware/pull/3686 travis build.

- [ ] don't forget a cherry-pick for MK3_3.12

### CODE REVIEW

File: lang/po/Firmware_nl.po [hunk 1]
```diff
@@ -561,7 +561,7 @@ msgstr "F. autoladen"
 #: ../../Firmware/messages.cpp:46 ../../Firmware/ultralcd.cpp:4232
 #: ../../Firmware/ultralcd.cpp:4239
 msgid "F. jam detect"
-msgstr ""
+msgstr "F.jam ontdekt"
```
Comment (vintagepc) [inline lang/po/Firmware_nl.po:564]: Is this phrase for indicating a jam has been detected or is it for a feature setting of "jam detection"?  Right now it looks like the tense disagrees

Comment (3d-gussner) [reply lang/po/Firmware_nl.po:564]: It is only for the LCD toggle menu to en-/disable it.

Comment (vintagepc) [reply lang/po/Firmware_nl.po:564]: right, but "ontdekt" sounds like the past tense of "it has been discovered". I'm suggesting we need a "present tense" "discovery" for this.

Comment (vintagepc) [reply lang/po/Firmware_nl.po:564]: "ontdekking" is probably too long, but I'd be happy just removing the t so it says "F. Jam ontdek."

**Review by vintagepc — APPROVED** [review]

**Comment by 3d-gussner** [comment]
Close this one in favor of a new PR.

**Pull request closed without merging**